src/odemis/gui/model/__init__.py

```diff
@@ -1385,10 +1389,11 @@ def getStreams(self):
         Do not modify directly, use addStream(), and removeStream().
         Note: use .stream_tree for getting the raw StreamTree (with the DataProjection)
         """
-        ss = self.stream_tree.getStreams()
+        ss = self.stream_tree.getProjections()
         # ss is a list of either Streams or DataProjections, so need to convert
         # back to only streams.
-        return [s.stream if isinstance(s, DataProjection) else s for s in ss]
+        # return [s.stream if isinstance(s, DataProjection) else s for s in ss]
```

Comment:
You can put back the original line, or, if you prefer, just remove the original line, and leave your new style.